tdrs-frontend/src/selectors/auth.js

```diff
@@ -56,12 +56,18 @@ export const accountIsMissingAccessRequest = (state) =>
 
 export const accountCanViewAdmin = (state) =>
   accountStatusIsApproved(state) &&
-  ['Developer', 'OFA System Admin', 'ACF OCIO', 'OFA Admin'].includes(
-    selectPrimaryUserRole(state)?.name
-  )
+  [
+    'Developer',
+    'OFA System Admin',
+    'ACF OCIO',
+    'OFA Admin',
+    'DIGIT Team',
```

Comment:
this enables the `Admin` tab on the frontend if the user has `DIGIT Team` as a role - this is a little weird when the user does not have superuser or staff privileges (though i think that problem pre-exists the digit team role specifically) - just want to clarify if digit team members are expected to have admin privileges

Reply:
They are expected to have Admin privileges in the same vain as `ACF OCIO`. The digit team can only see a very small subset of the entire admin console. Specifically, datafiles, datafile summary, and parser errors. I can double check with Alex if she wants all three or strictly just datafiles.

Reply:
Just re-read the requirements of the file, and the team needs to see the parsed data (duh haha) so I am going to add view permission for that also.